===================================================================
RCS file: /srv/anoncvs.freedesktop.org/cvs/gstreamer/gst-python/gst/gsttaglist.override,v
rcsdiff: /srv/anoncvs.freedesktop.org/cvs/gstreamer/gst-python/gst/gsttaglist.override,v: warning: Unknown phrases like `commitid ...;' are present.
retrieving revision 1.4
retrieving revision 1.5
diff -u -r1.4 -r1.5
--- gsttaglist.override	2005/12/18 17:20:12	1.4
+++ gsttaglist.override	2006/03/06 16:24:53	1.5
@@ -84,8 +84,6 @@
 		PyErr_SetString(PyExc_KeyError, field);
 	}
 		
-	if (v != NULL)
-		Py_INCREF(v);
 	return v;
 }
===================================================================
RCS file: /srv/anoncvs.freedesktop.org/cvs/gstreamer/gst-python/gst/gststructure.override,v
rcsdiff: /srv/anoncvs.freedesktop.org/cvs/gstreamer/gst-python/gst/gststructure.override,v: warning: Unknown phrases like `commitid ...;' are present.
retrieving revision 1.10
retrieving revision 1.11
diff -u -r1.10 -r1.11
--- gststructure.override	2006/01/08 12:26:35	1.10
+++ gststructure.override	2006/03/06 16:24:53	1.11
@@ -193,8 +193,6 @@
 		PyErr_SetString(PyExc_KeyError, field);
 	}
 		
-	if (v != NULL)
-		Py_INCREF(v);
 	return v;
 }
 

